pkg/compare/compare_test.go

```diff
@@ -329,6 +329,16 @@ error code:2`),
 			outputFormat: Json,
 			checks:       defaultChecks,
 		},
+		{
+			name:   "Allow Undefined Extras",
```

Comment:
change name to Check Merging ...... (not sure what is exactly tested here, maybe: only specified diffs are shown, in a different phrasing )  

also because we changed the name of the flag this should change to